install_all.js

```diff
@@ -4,6 +4,7 @@ const spawnSync = require('child_process').spawnSync;
 const commands = [
     { command: 'cd ./ilc/ && npm i', name: 'ilc' },
     { command: 'cd ./registry/ && npm i', name: 'registry' },
+    { command: 'cd ./registry/client && npm i', name: 'registry' },
```

Comment:
let's rename to eg `name: 'registry_admin'`

Reply:
Renamed to registry:ui to be consistent with `start_all.js`

Reply:
Renamed to registry:ui to be consistent with start_all.js